dart/lib/vm_service_lib.dart

```diff
@@ -272,8 +272,8 @@ class VmService {
       } else {
         _log.severe('unknown message type: ${message}');
       }
-    } catch (e) {
-      _log.severe('unable to decode message: ${message}, ${e}');
+    } catch (e, s) {
+      _log.severe('unable to decode message: ${message}, ${e}\n${s}');
```

Comment:
That API does not appear to exist, but we can add it and update this call. I agree that would be cleaner.